app/default/data.py

```diff
@@ -121,6 +121,23 @@ def get_round_data(fund_id, round_id, as_dict=False):
         return round_response
 
 
+def get_round_data_by_short_names(fund_short_name, round_short_name):
+    request_url = Config.GET_ALL_ROUNDS_FOR_FUND_ENDPOINT.format(
+        fund_id=fund_short_name
+    )
+    response = get_data(
+        request_url, {"language": get_lang(), "use_short_name": "true"}
+    )
+    return next(
+        (
+            round
+            for round in response
+            if str.upper(round["short_name"]) == str.upper(round_short_name)
+        ),
+        None,
```

Comment:
Ah, is `None` a default? That's pretty cool, however it might just defer to a `KeyError` exception:

https://github.com/communitiesuk/funding-service-design-frontend/blob/a572e6c033fa953b0b8acfeda57c69c3cdfc47e3/app/default/content_routes.py#L29

```bash
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
KeyError: 'title'
```

Don't know if this will ever happen though, so not sure how valuable it is to make the error more verbose... I think with the microservices architecture we're using there are quite a lot of scenarios like this due to the high amount of requests between services.  Thought I'd mention anyways since it caught my eye, but not a big deal.

Reply:
Thanks - added a check for round being None